minerl/herobraine/hero/mission.xml.j2

```diff
@@ -60,6 +60,11 @@
             {{ x }}
             {% endfor %}
 
+            <!--  Monitors     -->
+
+            {% for x in get_consolidated_xml((monitors)) %}
+            {{ x }}
+            {% endfor %}
```

Comment:
Oh interesting -- does this mean that Monitors were not previously activated on `dev`, if they were parts of EnvSpecs?

Reply:
Yep! But they were processed if they shared obs so)